src/Symfony/Component/Messenger/Bridge/MongoDB/composer.json

```
@@ -0,0 +1,32 @@
+{
+    "name": "symfony/mongodb-messenger",
+    "type": "symfony-messenger-bridge",
+    "description": "Symfony MongoDB Messenger Bridge",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.4",
+        "ext-mongodb": "^1.17|^2.0",
+        "mongodb/mongodb": "^1.17|^2.0",
```

We might drop support for old versions, including 1.x.
```suggestion
        "ext-mongodb": "^1.21|^2.0",
        "mongodb/mongodb": "^1.21|^2.0",
```

---

1.17 is the first supporting PHP 8.4, that's why I left it there. I would not expect much maintenance effort in leaving it as such.

Instead, I'm expecting a very slow adoption of 2.0 for now. It will be required for 8.5, so we may drop it in Symfony 9, while bumping our PHP requirement.